src/cgame/cg_main.c

```diff
@@ -641,6 +642,7 @@ static cvarTable_t cvarTable[] =
 	{ &cg_drawSpawnpoints,          "cg_drawSpawnpoints",          "0",           CVAR_ARCHIVE,                 0 },
 
 	{ &cg_useCvarCrosshair,         "cg_useCvarCrosshair",         "1",           CVAR_ARCHIVE,                 0 },
+	{ &cg_crosshairSVG,             "cg_crosshairSVG",             "0",           CVAR_ARCHIVE,                 0 },
```

Comment:
This needs to be a `CVAR_LATCH` as the shaders need to be re-registered to switch between TGA/SVG.

Reply:
Thanks! Done